apps/aecore/src/aec_parent_connector.erl

```diff
@@ -132,31 +112,20 @@ fetch_block_by_hash(Hash) ->
 fetch_block_by_height(Height) ->
     gen_server:call(?SERVER, {fetch_block_by_height, Height}).
 
-post_commitment(Who, Hash) ->
-    gen_server:call(?SERVER, {post_commitment, Who, Hash}).
-
 %%%=============================================================================
 %%% Gen Server Callbacks
 %%%=============================================================================
 
 -spec init([any()]) -> {ok, #state{}}.
-init([ParentConnMod, FetchInterval, ParentHosts, NetworkId, SignModule, HCPCPairs, Recipient, Fee, Amount]) ->
-    if is_integer(FetchInterval) ->
-        erlang:send_after(FetchInterval, self(), check_parent);
-        true -> ok
-    end,
-    CDetails =
-        #commitment_details{ parent_network_id  = NetworkId, %% TODO: assert all nodes are having the same network id
-                             sign_module = SignModule,
-                             recipient = Recipient,
-                             amount = Amount,
-                             fee = Fee
-                            },
+init([ParentConnMod, FetchInterval, ParentHosts, _NetworkId, _SignModule, HCPCPairs, _Recipient, _Fee, _Amount]) ->
```

Comment:
Are you expecting to use these arguments for pinning or could they be removed completely?

Reply:
I must admit I was a bit scared to remove them completely, but as that was last week I might have gathered up some courage now. Will take a look again then...